docs/using-hive.md

```diff
@@ -1191,8 +1191,24 @@ spec:
     name: pull-secret
 ```
 
+If the cluster you are looking to adopt is on AWS and leverages Privatelink, you'll also need to include that setting under `spec.platform.aws` to ensure the VPC Endpoint Service for the cluster is tracked in the ClusterDeployment
+
+```yaml
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: my-aws-cluster-creds
+      privateLink:
+        enabled: true
+      region: us-east-1
+```      
+
+It's also important to include any other configurations that may be in place in the original clusters' ClusterDeployment, such as `certificateBundles` settings or `controlPlaneConfig` settings as these are leveraged by certman-operator if in use. It's a good idea to compare your adoption manifest to your existing ClusterDeployment manifest. Any new secret refs added to your adoption manifest will require correlating secrets to live on the adopting cluster in the correct namespace.
```

Comment:
```suggestion
It's also important to include any other configurations that may be in place in the original cluster's ClusterDeployment, such as `certificateBundles` settings or `controlPlaneConfig` settings as these are leveraged by certman-operator if in use. It's a good idea to compare your adoption manifest to your existing ClusterDeployment manifest. Any new secret refs added to your adoption manifest will require correlating secrets to live on the adopting cluster in the correct namespace.
```

Hmm, couple things bugging me here:
- certman-operator is an OSD thing, and wouldn't be known to anyone outside of SRE-P (even OSD customers won't usually be aware of it).
- A discussion of adoption shouldn't really talk about an "existing ClusterDeployment manifest". If you have one of those, you should be using relocation instead of adoption.

Can we find a way to phrase all of this so it doesn't reference either of those things?

Reply:
yeah good points, as I was writing it I was worried this was too focused on OSD, and not considering OCP users who may be leveraging Hive. I could see that confusing non-OSD users. I'll work on cleaning that up a bit. Our scenario was a more unique case, and most probably won't be in that scenario.

Reply:
updated to only make note about privatelink as the other settings are not really covered in other places, and probably hyper specific to OSD. Keeping it simple.